consul/assets/saved_views/consul_errors.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Consul errors",
+    "type": "logs",
+    "page": "stream",
+    "query": "source:consul status:error",
```

Comment:
in other integration we also have an "overview" one that just `"query": "source:consul",`, do you think we should add it too?